apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/MockBigtableClient.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Bigtable.V2.Tests
+{
+    public class RequestMadeException : Exception { }
+
+    public class MockBigtableClient : BigtableClient
```

Comment:
I think you should be able to use Moq for this if it's easier to do so, but I don't mind a hand-rolled mock either.

Reply:
You might be right. Added a TODO to look into this later.